guides/integration-release.md

```diff
@@ -18,19 +18,21 @@ Don't do a release without any usage purpose, for example for the update of a te
 
 Follow the steps in the `Release Process` section in the `CONTRIBUTING.md` file of the repository carefully. These steps should be precise; if it's not, please open an issue on the related repository.
 
-During these steps, you might publish a release via the GitHub interface and so provide a description with the release changelogs.<br>
-The [Release Drafter tool](./release-drafter.md), integrated into most repositories, helps you provide a kind of structure, but please, feel free to add final modifications to ensure these changelogs are:
+#### Writing the Release Description
+
+During these release steps, you might publish a release via the GitHub interface and so provide a description with the release changelogs.<br>
```

Comment:
If the release is dont not via the GitHub interface, you cannot provide a description with the changelogs or is there no description?

Not sure I understand

Reply:
Not sure to understand your question 😂

Reply:
😂 
> you might publish a release via the GitHub interface and so provide a description with the release changelogs.

Can you not provide a description using de github CLI?

Reply:
This process is not for releases that don't use the GH interface. The integrations that need to release in a different way already have a really detailed section in their CONTRIBUTING (the devops integration repositories for example).

Reply:
Only the repositories that need to release to the GH interface link this guide in their CONTRIBUTING

Reply:
```suggestion
During these release steps, depending on the integration, releasing is done via the GitHub interface. The release comes with a description containing the changelogs. <br>
```